modules/simple_forms_api/app/form_mappings/vba_21_0845.json.erb

```diff
@@ -39,7 +39,7 @@
   "F[0].Page_2[0].ADDRESS_OF_PERSON_ZIPOrPostalCode_FirstFiveNumbers[0]": "<%= data.dig('person_address', 'postal_code')&.[](0..4) %>",
   "F[0].Page_2[0].ADDRESS_OF_PERSON_ZIPOrPostalCode_LastFourNumbers[0]": "<%= data.dig('person_address', 'postal_code')&.[](5..8) %>",
 
-  "F[0].Page_2[0].International_Telephone_Number_If_Applicable[1]": "<%= data['organization_name']&.+('\n')&.+(data['organization_representatives']&.map { |rep| rep['representative_name'] }&.join(', ')) %>",
+  "F[0].Page_2[0].International_Telephone_Number_If_Applicable[1]": "<%= (data['organization_name'] || '') + '\n' + (data['organization_representatives']&.map { |rep| rep['representative_name'] }&.join(', ') || '') %>",
```

Comment:
does it matter if we have the extra \n here when the one or more values are empty?